sigstore/root-signing
# sign-delegations for asraa

Signed-off-by: Asra Ali <asraa@google.com>



#### Summary

To verify, run
```
REPO=$(pwd)/ceremony/2022-07-12 ./scripts/verify.sh 319
```

You should see in the STAGED METADATA section, that the delegations (staging, rekor, revocation) are fully signed, and that root and targets are too.

#### Release Note


#### Documentation
